src/mono/mono/mini/aot-runtime.h

```diff
@@ -287,7 +287,7 @@ MonoAotMethodFlags mono_aot_get_method_flags (guint8 *code);
 typedef guint32 (*MonoAotResolvePltInfoOffset)(gpointer amodule, guint32 plt_entry_index);
 #endif
 
-#ifdef HOST_WASM
+#ifdef TARGET_WASM
```

Comment:
I don't understand this change.  These functions are needed at runtime, but not needed by the AOT  compiler - `HOST_WASM` seems correct for that case

Reply:
right, I changed the wrong define. fixed.